CHANGES.rst

```diff
@@ -267,6 +267,8 @@ Other Changes and Additions
 
 - Update bundled expat to 2.2.6. [#8343]
 
+- Added instructions for uploading releases to Zenodo. [#8395]
```

Comment:
why the changelog?

Reply:
I guess it doesn't hurt for those interested in citation.